package.json

```diff
@@ -21,7 +21,7 @@
     "precommit": "yarn git-hook",
     "prepush": "yarn git-hook && yarn snyk test",
     "start": "node app.js",
-    "start-watch": "yarn brunch watch & nodemon app.js | ./node_modules/bunyan/bin/bunyan",
+    "start-watch": "node headerItems.js & yarn brunch watch & nodemon app.js | ./node_modules/bunyan/bin/bunyan",
```

Comment:
This should be a double ampersand here as it means it must run and be successful.

Reply:
Duplicate comment...